pcli/src/command/stake.rs

```diff
@@ -25,10 +25,7 @@ pub enum StakeCmd {
     },
     /// Withdraw stake from a validator's delegation pool.
     Undelegate {
-        /// The identity key of the validator to withdraw delegation from.
-        #[structopt(long)]
-        from: String,
```

Comment:
I removed this since it turns out it's already implied by the delegation token denomination.  Alternatively, we could bring it back as an optional parameter, and allow the amount to be specified in either unit (i.e., i can just give the amount of delegation tokens to cash in, or, i can name a validator and an amount of unbonded stake, and have it compute the amount of delegation tokens).